src/main/java/com/google/devtools/build/lib/bazel/repository/starlark/StarlarkRepositoryModule.java

```diff
@@ -98,6 +97,13 @@ public StarlarkCallable repositoryRule(
         builder.addAttribute(attrDescriptor.build(attrName));
       }
     }
+    if (thread.getSemantics().experimentalRepoRemoteExec()) {
+      // Add the 'exec_properties' attribute last. That way it's backwards compatible with
+      // repository rules that already declare 'exec_properties'. In particular
+      // https://github.com/bazelbuild/bazel-toolchains
+      // TODO(buchgr): Remove this backwards compatiblity hack once the feature is marked stable.
```

Comment:
I don't think buchgr is the right person for this anymore. Probably should be me or @juliexxia